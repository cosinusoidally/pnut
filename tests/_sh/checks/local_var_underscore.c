// expect_comp_failure
void main() {
  int _a;
}
