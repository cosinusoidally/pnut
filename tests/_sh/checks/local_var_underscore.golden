_a tests/_sh/checks/local_var_underscore.c:5:0  variable name is invalid. It can't start or end with '_'.
